iOS/DuckDuckGoTests/LaunchOptionsHandlerTests.swift

```diff
@@ -35,37 +35,36 @@ final class LaunchOptionsHandlerTests: XCTestCase {
         try super.tearDownWithError()
     }
 
-    // MARK: - isUITesting
+    // MARK: - isOnboardingCompleted
 
-    func testShouldReturnTrueWhenIsUITestingIsCalledAndLaunchArgumentContainsIsUITesting() {
+    func testShouldReturnTrueWhenIsOnboardingCompletedAndEnvironmentsOnboardingIsTrue() {
         // GIVEN
-        let launchArguments = ["isUITesting"]
-        let sut = LaunchOptionsHandler(launchArguments: launchArguments, userDefaults: userDefaults)
+        let environment = ["ONBOARDING": "true"]
+        let sut = LaunchOptionsHandler(launchArguments: [], environment: environment, userDefaults: userDefaults)
 
         // WHEN
-        let result = sut.isUITesting
+        let result = sut.isOnboardingCompleted
 
         // THEN
         XCTAssertTrue(result)
     }
 
-    func testShouldReturnFalseWhenIsUITestingIsCalledAndLaunchArgumentsDoesNotContainIsUITesting() {
+    func testShouldReturnFalseWhenIsOnboardingCompletedAndEnvironmentsOnboardingIsFalse() {
```

Comment:
```suggestion
    func testShouldReturnFalseWhenIsOnboardingCompletedAndEnvironmentsOnboardingIsTrue() {
```

We should flip the “ONBOARDING”. similar to the above test